### PR TITLE
Add activity tags.

### PR DESCRIPTION
**Pull request by igor-tkachev**
#4561

### CODE REVIEW

**Review by viceroypenguin — APPROVED** [review]

File: Source/LinqToDB/Tools/ActivityTagID.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	public enum ActivityTagID
+	{
+		None = 0,
+		/// <summary>Database configuration name (connection string name).</summary>
+		ConfigurationString,
+		/// <summary>Data Provider name.</summary>
+		DataProviderName,
+		/// <summary>The name of the database server to which to connect.</summary>
+		DataSourceName,
+		/// <summary>The name of the current database.</summary>
```
Comment (MaceWindu) [inline Source/LinqToDB/Tools/ActivityTagID.cs:17]: Probably makes sense to specify that this information provided by provider as not all of them return data user expect :)

**Review by MaceWindu — APPROVED** [review]
except couple of minor notes, looks good

**Pull request merged**